openedx/core/djangoapps/discussions/models.py

```diff
@@ -609,6 +609,10 @@ class DiscussionTopicLink(models.Model):
         default=True,
         help_text=_("Whether this topic should be shown in-context in the course.")
     )
+    ordering = models.PositiveIntegerField(
+        null=True,
+        help_text=_("Ordering of this topic in its learning context"),
+    )
```

Comment:
how this field will be used? e.g. if there are 10 topics in a course, this will be the number in which the course topics will be ordered? 

Followup question: Can the topics be rearranged?

Reply:
When iterating over the units in a course, this value will get a number assigned based on its oder in the course. So the first unit might have an ordering of 101, the second one 102, and so on. 

The topics cannot be rearranged, since they are now explicitly tied to the course. The course units will need to be rearranged to rearrange these topics. 

However, they can be sorted in the frontend.